docs/installation/manage/_index.md

```diff
@@ -260,6 +260,49 @@ cat $(brew --prefix)/var/log/viam.log
 {{% /tab %}}
 {{< /tabs >}}
 
+## Uninstall `viam-server`
+
+{{< tabs name="Uninstall viam-server">}}
+{{% tab name="Linux"%}}
+
+Remove the system installed service with the following three commands:
+
+```sh {class="command-line" data-prompt="$"}
+sudo systemctl disable --now viam-server
+sudo rm /etc/systemd/system/viam-server.service
+sudo systemctl daemon-reload
+```
+
+To remove various Viam caches and logs for the root (service) user, run:
+
+```sh {class="command-line" data-prompt="$"}
+sudo rm -r /root/.viam/
+```
+
+If you ever run `viam-server` directly, to remove various Viam caches and logs for your own user run:
+
+```sh {class="command-line" data-prompt="$"}
+rm -r ~/.viam/
+```
+
+To remove the `viam-server` binary itself, run:
+
+```sh {class="command-line" data-prompt="$"}
+sudo rm /usr/local/bin/viam-server
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+
+Uninstall `viam-server` with the following command:
+
+```sh {class="command-line" data-prompt="$"}
+brew uninstall viam-server
```

Comment:
This may also need ~/.viam and /etc/viam.json removed with the same command as for linux.

Reply:
Done I think, is this what you meant with both possibilities/commands?